aodntools/timeseries_products/velocity_aggregated_timeseries.py

```diff
@@ -0,0 +1,317 @@
+import os
+import sys
+import tempfile
+import shutil
+import netCDF4 as nc4
+import numpy as np
+import json
+from datetime import datetime
+import argparse
+from pkg_resources import resource_filename
+from aodntools import __version__
+
+import xarray as xr
+
+import aggregated_timeseries as utils
```

Comment:
This may work when you run the code in your terminal, but it won't work in general. Please specify the full path within the package:
```suggestion
from aodntools.timeseries_products import aggregated_timeseries as utils
```

Reply:
changed